File: CheckforString.java
public class CheckforString {
    
public static boolean areStringsSame(String s1 , String s2){
    return s1.equals(s2);
}

public static void main(String [] args ){
    String s1 = "abc";
    String s2 = "abcd";

    if(areStringsSame(s1 , s2)){
        System.out.println("yes");
    }else{
        System.out.println("No");
    }
  }

}

File: CheckforString.c
#include<stdio.h>
#include<string.h>
#include<stdbool.h>


bool areStringsSame( const char* s1 , const char* s2){
    return strcmp(s1 , s2) == 0;
}

int main()
{
    const char* s1 = "abc";
    const char* s2 = "abcd";


    if(areStringsSame(s1 , s2)){
        printf("Yes\n");
    } else {
        printf("No\n");
    }

    return 0;
}